reference/composedcubemaptexture.md

```diff
@@ -0,0 +1,126 @@
+## ComposedCubeMapTexture
+
+```
+ComposedCubeMapTexture {
+  vrmlField SFNode right   NULL
+  vrmlField SFNode left    NULL
+  vrmlField SFNode top     NULL
+  vrmlField SFNode bottom  NULL
+  vrmlField SFNode front   NULL
+  vrmlField SFNode back    NULL
+}
+```
+
+### Description
+
+The [ComposedCubeMapTexture](#composedcubemaptexture) node defines a cubic texture used for environment mapping. It is composed of exactly six [ImageTexture](imagetexture.md) nodes. If one of the [ImageTexture](imagetexture.md)s is not set, the corresponding side of the cubic texture will be black.
```

Comment:
This node actually does deal with environment mapping specifically, much like the corresponding X3D node does: http://doc.x3dom.org/author/CubeMapTexturing/ComposedCubeMapTexture.html
Furthermore, the texture is mirrored such that it is convenient for reflection mapping, and not for basic texturing.
Therefore I think it is worth mentioning early.